app/pages/AboutPageOur415/AboutPage.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useEffect, useState } from "react";
+import { SanityImageSource } from "@sanity/image-url/lib/types/types.d";
+import { client } from "../../sanity";
+import { Masthead } from "../../components/ui/Masthead/Masthead";
+import { EmailSignup } from "../../components/EmailSignup/Emailsignup";
+import { TwoColumnContentSection } from "../../components/ui/TwoColumnContentSection/TwoColumnContentSection";
+
+interface TwoColumnContentSection {
+  mediaAlignment: string;
+  image: SanityImageSource;
+  imageAlt: string | undefined;
+  contentBlock: any;
+  contentLinkButtonText: string;
+  contentLinkButtonUrl: string;
+  _id: string;
+}
+
+interface PageState {
+  pageInitialized: boolean;
+  mastHead: string;
+  twoColumnContentSections: TwoColumnContentSection[];
+}
+
+export const AboutPage = () => {
+  const [pageData, setPageData] = useState<PageState>({
```

Comment:
This is nice, having just one state. I will keep this in mind so we are using uniform conventions across components/pages.